tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,112 @@ describe('Issue model class', () => {
+        phaseTeamResponseIssue.teamResponse = 'Sample Text';
+        expect(phaseTeamResponseIssue.createGithubTeamResponse())
+            .toEqual(`# Team\'s Response\n${phaseTeamResponseIssue.teamResponse}\n ## Duplicate status (if any):\n--`);
+
+        const phaseTeamResponseIssue2 = dummyIssue.clone(Phase.phaseTeamResponse);
+        phaseTeamResponseIssue2.teamResponse = 'Sample Text';
+        phaseTeamResponseIssue2.duplicateOf = 10;
+        expect(phaseTeamResponseIssue2.createGithubTeamResponse())
+            .toEqual(`# Team\'s Response\n${phaseTeamResponseIssue2.teamResponse}\n `
+                + `## Duplicate status (if any):\nDuplicate of #${phaseTeamResponseIssue2.duplicateOf}`);
+    });
+
+    it ('should be able to get the proper Tutor Response', () => {
+        const phaseModerationIssue = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        expect(phaseModerationIssue.createGithubTutorResponse()).toEqual(tutorResponseStringHeader);
+
+        const phaseModerationIssue2 = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        phaseModerationIssue2.issueDisputes = [newIssueDispute];
+        expect(phaseModerationIssue2.createGithubTutorResponse()).toEqual(tutorResponseStringHeader
+            + newIssueDispute.toTutorResponseString());
+    });
+
+    it ('should be able to get the proper Tester Response', () => {
+        const phaseTesterResponseIssue = dummyIssueWithTeam.clone(Phase.phaseTesterResponse);
+        phaseTesterResponseIssue.teamResponse = 'Sample Text';
+        phaseTesterResponseIssue.testerResponses = [];
+        expect(phaseTesterResponseIssue.createGithubTesterResponse()).toEqual(
+            `# Team\'s Response\n${phaseTesterResponseIssue.teamResponse}\n ` +
+        `# Items for the Tester to Verify\n${''}`);
+
+        const phaseTesterResponseIssue2 = dummyIssueWithTeam.clone(Phase.phaseTesterResponse);
+        phaseTesterResponseIssue2.teamResponse = 'Sample Text';
+        phaseTesterResponseIssue2.testerResponses = [newTesterResponse];
+        expect(phaseTesterResponseIssue2.createGithubTesterResponse()).toEqual(
+            `# Team\'s Response\n${phaseTesterResponseIssue.teamResponse}\n ` +
+        `# Items for the Tester to Verify\n${newTesterResponse.toString()}`);
+    });
+
+    it ('returns the correct number of issue disputes', () => {
+        const phaseModerationIssue = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        expect(phaseModerationIssue.numOfUnresolvedDisputes()).toEqual(0);
+
+        const phaseModerationIssue2 = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        phaseModerationIssue2.issueDisputes = [newIssueDispute];
+        expect(phaseModerationIssue2.numOfUnresolvedDisputes()).toEqual(1);
+
+        const phaseModerationIssue3 = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        phaseModerationIssue3.issueDisputes = [newIssueDispute];
+        newIssueDispute.todo = '- [x] Done' ;
```

Comment:
Instead of setting `newIssueDispute.todo` directly, can we use the `setIsDone` method exposed by `IssueDispute` ?
i.e. `newIssueDispute.setIsDone(true)` ?